### PR TITLE
Feat/deploy doc

### PR DESCRIPTION
**Pull request by raksiv**
Nitric users have commented that this should be called out as a page of its own for quick reference.

### CODE REVIEW

File: doc.config.json [hunk 1]
```diff
@@ -23,6 +23,10 @@
                 "title": "Getting Started",
                 "path": "/docs/getting-started.mdx"
               },
+              {
+                "title": "Deploying",
+                "path": "/docs/deploy-your-app.mdx"
+              },
```
Comment (tjholm) [inline doc.config.json:29]: Dfeels a bit odd to have `Deploying` and `Deployment` as two pages in the same menu. Is there a way we could merge this content?

**Review by tjholm — CHANGES_REQUESTED** [review]

File: doc.config.json (hunk 1 above)
Comment (raksiv) [reply doc.config.json]: Agreed, I don;t think merging it is the right solution.
We want the deploy steps to be short and easily reachable - I've moved it into the guides section and renamed it to Deploying with the CLI

**Review by asalkeld — APPROVED** [review]

File: docs/deploy-your-app.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```
+? What do you want to call your new stack? dev
+? Which Cloud do you wish to deploy to? aws
+? select the region us-east-1
+```
+
+Now you can deploy your stack named `dev` with the `up` command.
+
+```bash
+nitric up -s dev
```
Comment (HomelessDinosaur) [inline docs/deploy-your-app.mdx:31]:
```suggestion
nitric up
```

File: docs/deploy-your-app.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+| main | https://XXXXXXXX.execute-api.us-east-1.amazonaws.com |
+└───────────────────────────────────────────────────────────────┘
+```
+
+When the deployment is complete, go to the relevant cloud console and you'll be able to see and interact with your API.
+
+To undeploy this stack -
+
+```bash
+nitric down -s dev
```
Comment (HomelessDinosaur) [inline docs/deploy-your-app.mdx:52]:
```suggestion
nitric down
```

File: docs/deploy-your-app.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Follow the prompts to create a stack for your provider, this example will use `aws`.
+
+```
+? What do you want to call your new stack? dev
+? Which Cloud do you wish to deploy to? aws
+? select the region us-east-1
+```
+
+Now you can deploy your stack named `dev` with the `up` command.
```
Comment (HomelessDinosaur) [inline docs/deploy-your-app.mdx:28]:
```suggestion
Now you can deploy your stack with the `up` command.
```

**Review by HomelessDinosaur — CHANGES_REQUESTED** [review]
We can remove `-s` from the up and down commands now

**Pull request closed without merging**